documentation/configuration.markdown

```diff
@@ -75,9 +79,9 @@ This file contains the settings for Puppet Server itself.
 # configuration for the JRuby interpreters
 
 jruby-puppet: {
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/puppet/cache/jruby-gems
```

Comment:
Not sure if this one is final, but I couldn't find this listed at https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md.  Would be good to sync this up with that doc....